CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # HDMF Changelog
 
+## HDMF 3.5.7 (Upcoming)
```

Comment:
```suggestion
## HDMF 3.6.0 (Upcoming)
```

Since this is not a feature release and not a bug fix, we should bump the minor version.